PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -29,12 +29,14 @@
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
+    <NoWarn>CS2008</NoWarn>
   </PropertyGroup>
   <PropertyGroup>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <SourcesRoot>$(MSBuildThisFileFullPath)\..\..\..</SourcesRoot>
     <DestinationDir>$(SourcesRoot)\DeploymentArtifacts\CSharpPluginPayload\$(Configuration)\</DestinationDir>
     <!-- OutputPath is required by the Clean and Rebuild tasks -->
-    <OutputPath>$(DestinationDir)</OutputPath>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
```

Comment:
This is the actual fix for the packaging issue. The <OutputPath> needs to be set so that the project will build but it shouldn't be <DestinationDir>, which should only contain the files to be included in the zip.